### PR TITLE
Fixed display layout

### PR DESCRIPTION
**Pull request by aryanshridhar**
<ol>
<li>Fixed Items alignment</li>
<li>Added hover effects</li>
<li>Changed socialize section</li>
<li>Updated info till some extent </li>
</ol>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,173 +3,150 @@
...
-					<span class="icon">
-						<i class="fa fa-twitter"></i>
-					</span>
-				</a>
+		<!-- NAVBAR -->
+		<div class="container">
+		<nav class="navbar navbar-expand-sm">
+			<div class="move">
+				<a href ="https://medium.com/@ayush1999/" class ="fa fa-medium nav"></a>
+				<a href ='https://github.com/ayush1999' class ="fa fa-github nav"></a>
```
Comment (ashridh) [inline index.html]: `https://github.com/ayush-1506`

File: index.html [hunk 2]
```diff
@@ -3,173 +3,150 @@
...
+							- September'19: Successfully completed GSoC'19 with Mozilla. I've published a brief blog about my work on medium :
+							<a target = "_blank" href="https://medium.com/@ayush1999/gsoc19-mozilla-a185b66d9564">here</a>.<br><br>
+							- August'19 : Back in India! Heading to my university at the moment. It was a great experience
+							working in UCL. I hope to continue contributing as and when I get time.<br><br>
+							- May'19: Very rarely do I have more than one good news in a single month. I guess this is one of those rare
+							occasions. I got my talk selected in JuliaCon'19 in University of Maryland, Baltimore. It's great to get
+							this opportunity to showcase my work to a wide audience. Hopefully my visa gets approved (which is doubtful,
+							since I'll be applying from UK,which isn't my home country.). Let's see how this goes. <br><br>
+							- May'19 : Some good news: I got my Google Summer of Code project selected. This time, I'll be working under the 
+							Mozilla organization on the <i>bugbug</i> project. I'll be tough managing it with my project in UCL, let's
```
Comment (ashridh) [inline index.html]: nit: `I'll -> It'll`

**Review by ashridh — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (aryanshridhar) [reply index.html]: No href link provided for projects section in navbar.

Comment (ayush-1506) [reply index.html]: Link them to https://github.com/ayush-1506?tab=repositories for now.

Comment (ashridh) [reply index.html]: Replace projects with `Blog` in navbar. href for this should link to `blog/main.html`

File: index.html [hunk 3]
```diff
@@ -3,173 +3,156 @@
...
+				</span>
+			</button>
+		  
+			<div class="collapse navbar-collapse" id="navbarSupportedContent">
+			  <ul class="navbar-nav ml-auto">
+				<li class="nav-item active">
+				  <a class="nav-link cus" href="#about">About</a>
+				</li>
+				<li class="nav-item">
+				  <a class="nav-link cus" href="/blog/main.html">Blog</a>
```
Comment (ashridh) [inline index.html]: Test changes locally before committing. (This href doesn't exist. I believe this should be blog/index.html).

**Review by ashridh — CHANGES_REQUESTED** [review]

**Pull request merged**